modules/core/src/v2/keycard.ts

```diff
@@ -72,14 +72,15 @@ const generateQuestions = (coin: string) => {
         [
           'No! BitGo’s multi-signature approach to security depends on there not being a single point ',
           'of attack. But if your wallet password is on your KeyCard, then anyone who gains access to ',
-          'your KeyCard will be able to steal your ' + coin + '.' + ' We recommend keeping your wallet password ',
+          `your KeyCard will be able to steal your ${coin}. We recommend keeping your wallet password `,
           'safe in a secure password manager such as LastPass, 1Password or KeePass.'
         ]
     }
   ];
 };
 
 interface GetKeyDataOptions {
+  encrypt: (params: {input: string, password: string}) => any,
```

Comment:
This should return a string instead of `any`.

Reply:
Great point, not sure why I went with `any` there!